druid-shell/src/common_util.rs

```diff
@@ -21,7 +21,8 @@ use std::sync::atomic::{AtomicU64, Ordering};
 /// Strip the access keys from the menu string.
 ///
 /// Changes "E&xit" to "Exit". Actual ampersands are escaped as "&&".
-#[cfg(any(target_os = "macos", target_os = "linux"))]
+// TODO(x11/menus): Re-evaluate this #[cfg]; used to be just "macos OR linux" before.
+#[cfg(any(target_os = "macos", all(target_os = "linux", not(feature = "x11"))))]
```

Comment:
at this point it might just be easier to have a `#[allow(dead_code)]`. 😒

Reply:
haha, fair. Can do.

Reply:
Done.